shared/xccdf/system/logging.xml

```diff
@@ -12,12 +12,14 @@ In Red Hat Enterprise Linux 7, rsyslog has replaced ksyslogd as the
 syslog daemon of choice, and it includes some additional security features
 such as reliable, connection-oriented (i.e. TCP) transmission of logs, the
 option to log to database formats, and the encryption of log data en route to
-a central logging server.
+a central logging server.<br />
+In Debian and Ubuntu, rsyslog is on of the two main logservers, with syslog-ng,
```

Comment:
This shouldn't be added here. It should be in a separate description with `prodtype`